nixos/modules/services/web-apps/freshrss.nix

```diff
@@ -267,7 +267,7 @@ in
           Group = "freshrss";
           StateDirectory = "freshrss";
           WorkingDirectory = cfg.package;
-          ExecStart = "${pkgs.php}/bin/php ./app/actualize_script.php";
+          ExecStart = "./app/actualize_script.php";
```

Comment:
Somehow this change did break my updater... not sure why though.

systemd tells me this:
```
Oct 23 12:34:05 hostname systemd[1]: freshrss-updater.service: Cannot add dependency job, ignoring: Unit freshrss-updater.service has a bad unit file setting.
Oct 23 18:04:10 hostname systemd[1]: /etc/systemd/system/freshrss-updater.service:15: Neither a valid executable name nor an absolute path: ./app/actualize_script.php
Oct 23 18:04:10 hostname systemd[1]: freshrss-updater.service: Unit configuration has fatal error, unit will not be started.
```

And yes, the setting `WorkingDirectory=/nix/store/grx4l0y5imn5ib1f61g4rpcm9292lr0z-FreshRSS-1.20.0` is set in the unit and it has the `ExecStart=./app/actualize_script.php`.

When I inspect `/nix/store/grx4l0y5imn5ib1f61g4rpcm9292lr0z-FreshRSS-1.20.0/app/actualize_script.php` it is executable with the shebang `#!/nix/store/qcagvy6rrhaf9hkf0fykimcvgba92kdq-php-with-extensions-8.1.11/bin/php`

Then I tried to stand in the correct working directory and execute:
```
sudo -u freshrss FRESHRSS_DATA_PATH=/var/lib/freshrss ./app/actualize_script.php
```

And this... seems to work...

Reply:
Once I've read the error message again I agree with it... It's not an absolute path, it's a relative path. It's also not an executable name. So maybe we should just make `ExecStart = "${cfg.package}/app/actualize_script.php";` and it should be fine.

Reply:
i’m not sure why this doesn’t error on my machine… but you’re totally right: nowhere else in nixpkgs do we use a `./` relative path for ExecStart. i’ll throw up that one-liner patch as a PR in a bit here when i’m free, if nobody’s already done it.